<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1062</width>
    <height>694</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QHBoxLayout" name="horizontalLayout_7">
    <item>
     <widget class="QWidget" name="horizontalWidget" native="true">
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <layout class="QVBoxLayout" name="verticalLayout">
         <item>
          <widget class="QVTKWidget" name="axialWidget"/>
         </item>
         <item>
          <widget class="QVTKWidget" name="sagittalWidget"/>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QVBoxLayout" name="verticalLayout_2">
         <item>
          <widget class="QVTKWidget" name="coronalWidget"/>
         </item>
         <item>
          <widget class="QVTKWidget" name="threeDWidget"/>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="QWidget" name="verticalWidget" native="true">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Maximum" vsizetype="Preferred">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_3">
       <item>
        <widget class="QPushButton" name="load_Image_Btn">
         <property name="text">
          <string>Load Image</string>
         </property>
        </widget>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_3">
         <item>
          <widget class="QPushButton" name="load_Atlas_Btn">
           <property name="text">
            <string>Load Atlas</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="del_Atlas_Btn">
           <property name="text">
            <string>Del Atlas</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_5">
         <item>
          <widget class="QPushButton" name="config_Tracker_Btn">
           <property name="text">
            <string>Configure Tracker</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="sel_Tracker_Btn">
           <property name="text">
            <string>Select Tracker</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_4">
         <item>
          <widget class="QPushButton" name="sel_Marker_Btn">
           <property name="text">
            <string>Select Markers</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="cap_Marker_Btn">
           <property name="text">
            <string>Capture</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="ok_Sel_Btn">
           <property name="text">
            <string>Finish</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_6">
         <item>
          <widget class="QPushButton" name="start_Tracking_Btn">
           <property name="text">
            <string>Start Tracking</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="stop_Tracking_Btn">
           <property name="text">
            <string>Stop Tracking</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_2">
         <item>
          <widget class="QLabel" name="label">
           <property name="text">
            <string>Opacity</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QSlider" name="opacity_Slider">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1062</width>
     <height>30</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QVTKWidget</class>
   <extends>QWidget</extends>
   <header>QVTKWidget.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
